lib/rules/no-only.js

```diff
@@ -16,6 +16,7 @@ const utils = require("../utils");
 
 module.exports = {
     meta: {
+        type: "problem",
```

Comment:
Should this one be a "suggestion"? It's a problem when committing code but only a suggestion when doing local development, hence why I used the "Best Practices" category.

Reply:
Sounds good, fixed!